deploy/handler/operator.yaml

```diff
@@ -223,3 +229,15 @@ webhooks:
         apiGroups: ["*"]
         apiVersions: ["v1alpha1"]
         resources: ["nodenetworkconfigurationpolicies", "nodenetworkconfigurationpolicies/status"]
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{template "handlerPrefix" .}}nmstate-webhook
+  namespace: {{ .HandlerNamespace }}
+  labels:
+    app: kubernetes-nmstate
+type: kubernetes.io/tls
+data:
+  tls.crt: YmFkIGNlcnRpZmljYXRlCg==
```

Comment:
This should be empty by default, or keeping some obvious placeholder.

Reply:
It cannot be empty and it has to be base64